sbom-convert.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: sbom-convert
+  version: 0.0.4
+  epoch: 0
+  description: CLI tool based on the protobom library that converts Software Bills of Materials across formats (SPDX and CycloneDX).
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/protobom/sbom-convert
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+
+  - uses: go/build
+    with:
+      output: sbom-convert
+      packages: .
+
+  - uses: strip
```

Comment:
We should also add an `update:` block here, which will tell Wolfi's automation when/how to update the package when 0.0.5+ comes out. You can find some examples throughout the repo, but it should be pretty simple since it's just on GitHub. Look for

```
update:
  enabled: true
  github:
    ... 
```

Reply:
Added the `update` block